packages/sdk/src/services/invitation-service.ts

```diff
@@ -91,7 +91,7 @@ export class InvitationService extends Service<InvitationApi> {
     );
 
     this.logger.log('Accepting invitation');
-    return this.vaultAPIFactory(vault_access_token)
+    return this.vaultAPIFactory({ vault_access_token })
```

Comment:
Nothing comes to mind as a way to ensure that the above doesn't re-occur as we really do want the delegationId property to be totally optional.

I have gone through and ensured that all the services are passing the full object (and that the property is called credentials consistently), so at the very least the convention is well set and hopefully copied.

Reply:
One way to do it would be sth like `vaultAPIFactory(credentials: IVaultToken, delegation: IDelegationToken | null)`, so every time you use api factory you have to pass the token separately, explicitly setting to null if it doesn't exist. But that trades convenience for safety, don't know if it's called for right now honestly.

Another way that would keep the same interface would be to add delegation credentials to the mock user credentials and test for its existence after each method. That would be a bit of work though, and we would probably do it when we get to implementing the full delegation flow in the SDK. Seems like something for a later sprint.